scripts/write-all-commands.js

```diff
@@ -0,0 +1,38 @@
+import fs from 'fs';
+import path from 'path';
+import url, { pathToFileURL } from 'url';
```

Comment:
Cant we just import the functions `fileURLToPath` and `pathToFileURL` instead of the entire default?